api/TimeAddressableMediaStore.yaml

```diff
@@ -1962,7 +2036,7 @@ paths:
         "400":
           $ref: '#/components/responses/trait_resource_info_head_400'
         "404":
-          description: The requested media object does not exist.
+          description: The requested media object does not exist. 404 MUST be returned if the ID has been assigned via the storage endpoint, but not yet registered against a segment.
```

Comment:
Not sure if this was deliberate or missing consistency. This extra note has not been added to the 404 response to the `GET` method.

Reply:
Good spot. Fixed.